File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        int min = 1000000;
        int max = -1000000;

        for(int i = 0; i<num; i++){
            int a = sc.nextInt();

            if(a>max)
                max = a;
            if(a<min)
                min = a;
        }
        System.out.println(min + " " + max);
    }
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main() {
    int a, min, max;
    scanf("%d", &a);

    int b[a];

    min = 1000000;
    max = -1000000;
    for (int i = 0; i < a; i++) {
        scanf("%d", &b[i]);
        if (b[i] > max)
            max = b[i];
        if (b[i] < min)
            min = b[i];
    }
    printf("%d %d", min, max);

    return 0;
}
